src/components/Table/GridTable.tsx

```diff
@@ -174,6 +174,7 @@ export type GridSortConfig<S> =
       /** The optional initial column (index in columns) and direction to sort. */
       initial?: [S | GridColumn<any>, Direction] | undefined;
       caseSensitive?: boolean;
+      persistent?: S | GridColumn<any> | undefined;
```

Comment:
I'd like to chat more about this,  

There isn't currently a need for it, however if there is a use case or you think it would just be better to keep it consistent I respect your wisdom.

currently I'm struggling to get this working as is, (working in beam and not in internal frontend)

I will circle back to this once I get things settled, unless you insist otherwise :)